### ISSUE TITLE
[Issue]: Various files fail using ffprobe "Input string was not in a correct format"

### ISSUE THREAD

**Issue by dionysius** [jellyfin/jellyfin#9026]
### Please describe your bug

I will need input from you on what you need to debug this. I understand crap media metadata input makes crap output.

### Jellyfin Version

10.8.4-1

### Environment

```markdown
- OS: Ubuntu jammy
- Virtualization: LXD
- FFmpeg Version: 5.1.1-1-jammy
- Storage: with lxd mounted disk
```

### Jellyfin logs

```shell
[2023-01-07 05:00:00.782 +00:00] [ERR] Extracting keyframes from "/mnt/samba/public/Movies/XXX/XXX.avi" using ffprobe failed
System.FormatException: Input string was not in a correct format.
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type)
   at Jellyfin.MediaEncoding.Keyframes.FfProbe.FfProbeKeyframeExtractor.ParseStream(StreamReader reader)
   at Jellyfin.MediaEncoding.Keyframes.FfProbe.FfProbeKeyframeExtractor.GetKeyframeData(String ffProbePath, String filePath)
   at Jellyfin.MediaEncoding.Hls.Extractors.FfProbeKeyframeExtractor.TryExtractKeyframes(String filePath, KeyframeData& keyframeData)
```
And this over and over again for the same and some other files.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cvium** [comment]
Have you tried upgrading? That bug was fixed in 10.8.6.

**Comment by dionysius** [comment]
Oh, yeah that makes this issue totally not worth looking at. Sorry, I should technically have auto-update... checking.
